test/accounts_test.go

```diff
@@ -15,6 +16,71 @@ import (
 	"github.com/checkout/checkout-sdk-go/nas"
 )
 
+func TestSubmitFileAccounts(t *testing.T) {
```

Comment:
Please reorder to maintain consistency with client

Reply:
👍